spec/services/middleware_topology_service_spec.rb

```diff
@@ -116,18 +160,36 @@
                                                                        :key          => "MiddlewareMessaging" + mw_messaging.compressed_id.to_s}
       )
 
-      expect(subject[:relations].size).to eq(5)
+      expect(subject[:relations].size).to eq(7)
       expect(subject[:relations]).to include(
-        {:source => "MiddlewareManager" + ems_hawkular.compressed_id.to_s,
-         :target => "MiddlewareServer" + mw_server.compressed_id.to_s},
-        {:source => "MiddlewareServer" + mw_server.compressed_id.to_s,
-         :target => "MiddlewareDeployment" + mw_deployment1.compressed_id.to_s},
-        {:source => "MiddlewareServer" + mw_server.compressed_id.to_s,
-         :target => "MiddlewareDeployment" + mw_deployment2.compressed_id.to_s},
-        {:source => "MiddlewareServer" + mw_server.compressed_id.to_s,
-         :target => "MiddlewareDatasource" + mw_datasource.compressed_id.to_s},
-        {:source => "MiddlewareServer" + mw_server.compressed_id.to_s,
-         :target => "MiddlewareMessaging" + mw_messaging.compressed_id.to_s}
+        {
+          :source => "MiddlewareManager" + ems_hawkular.compressed_id.to_s,
+          :target => "MiddlewareServer" + mw_server.compressed_id.to_s
+        },
+        {
+          :source => "MiddlewareManager" + ems_hawkular.compressed_id.to_s,
+          :target => "MiddlewareServerEap" + mw_server_eap.compressed_id.to_s
+        },
+        {
+          :source => "MiddlewareManager" + ems_hawkular.compressed_id.to_s,
+          :target => "MiddlewareServerWildfly" + mw_server_wildfly.compressed_id.to_s
+        },
+        {
+          :source => "MiddlewareServer" + mw_server.compressed_id.to_s,
```

Comment:
like above, I think all MiddlewareServer is not going to appear anymore, unless I got it wrong